app/views/liquids/_open_graph.html.erb

```diff
@@ -0,0 +1,33 @@
+<% if page.main_properties_present? %>
+  <div class="crayons-card p-5">
+    <% if page.image %>
+      <div class="s:max-w-50 mb-4">
+        <img src="<%= page.image %>" height="auto" alt="Cover image">
+      </div>
+    <% end %>
+    <div>
+      <a href="<%= page.url %>" target="_blank" rel="noopener noreferrer">
+        <h2 class="fs-xl fw-bold lh-tight mb-2">
+          <%= page.title %>
+        </h2>
+      </a>
+      <% if page.description %>
+        <p class="truncate-at-2 mb-2">
+          <%= page.description %>
+        </p>
+      <% end %>
+    </div>
+    <div class="color-secondary fs-s">
+      <% if page.site_name %>
+        <span class="fw-medium"><%= page.site_name %> &bull;</span>
+      <% end %>
+      <span>
+        <%= page.url %>
```

Comment:
This is more of a design question, but I couldn't tell from the Figma if the intention is to only show host/domain or to show the whole url - e.g. if given `https://www.theguardian.com/tv-and-radio/2022/mar/30/disgraceful-neglect-of-line-of-duty-the-wrongs-and-rights-of-the-bafta-tv-nominations` do we show that whole URL, or just `guardian.com`? 

I was just wondering since it can be pretty long (maybe one for @ludwiczakpawel to confirm)

Reply:
Yes I was thinking that as well. I believe showing the host only makes better sense, and we already have the title linked to the entire URL.